theano/sandbox/rng_mrg.py

```diff
@@ -1363,6 +1363,50 @@ def multinomial(self, size=None, n=1, pvals=None, ndim=None, dtype='int64',
             raise NotImplementedError(("MRG_RandomStreams.multinomial only"
                                        " implemented for pvals.ndim = 2"))
 
+    def weighted_selection(self, size=None, n=1, pvals=None, ndim=None, dtype='int64',
+                           nstreams=None):
+        """
+        Sample `n` times *WITHOUT replacement* from a multinomial distribution
+        defined by probabilities pvals. `n` needs to be in [1, m], where m is the number of
+        elements to select from, i.e. m == pvals.shape[1]. By default n = 1.
+        
+        Example : WRITEME
```

Comment:
I agree there should be an example. I'm just putting a comment so I don't forget to look for the example in the final version of your PR.